contrib/zpages/README.md

```diff
@@ -50,6 +50,37 @@ public class MyMainClass {
 }
 ```
 
+### View stats and spans on Z-Pages
+
+#### View RPC stats on /rpcz page
+
+The /rpcz page displays the canonical RPC stats broken down by RPC methods. Example:
```

Comment:
I think this section should mention that the stats are for gRPC.

Reply:
Done.